src/index.js

```diff
@@ -1,13 +1,13 @@
 // @ts-check
-import os from "os"
-import fs from "fs"
-import path from "path"
 import * as core from "@actions/core"
 import * as github from "@actions/github"
 import * as tc from "@actions/tool-cache"
 import { Octokit } from "@octokit/rest"
+import fs from "fs"
+import os from "os"
+import path from "path"
```

Comment:
Please refrain from mixing in such unrelated changes, it makes reviewing the changes unnecessarily harder.